src/biokbase/narrative/contents/kbasewsmanager.py

```diff
@@ -210,35 +210,35 @@ def get(self, path, content=True, type=None, format=None):
         path = path.strip('/')
         model = base_model(path, path)
         try:
-            if self.exists(path) and type != u'directory':
+            if self.exists(path) and type != 'directory':
```

Comment:
Shadows a built-in but probably beyond the refactor's scope

Reply:
Yeah, that should get changed. That module is inherited from a base class in the Jupyter stack that deals with file management, etc. I suspect that API's been updated along the way, too.

Probably a separate PR.